main.js

```diff
@@ -111,26 +111,55 @@ function dragoverHandler(event) {
 
 function dropHandler(event) {
   event.preventDefault();
-  var dragId = parseInt(event.dataTransfer.getData('color-box'));
+  var dragId = parseInt(event.dataTransfer.getData("color-box"));
+  var dropId = parseInt(event.target.dataset.id);
   var dropId = parseInt(event.target.parentNode.dataset.id);
```

Comment:
I noticed we have 2 declarations of var dropId here, do we need both?

Reply:
Idk how that got there. I don't think we need line 115. I'll fix it and push it.

I took off and tested it and as far as I can see everything works fine.